<?xml version="1.0"?>

<launch>
    <node pkg="cmd_filtered" type="cmd_filtered" name="cmd_filtered" output="screen" cwd="node">
        <param name="cmd_filtered/imu_topic_name" type="str" value="imu"/>

        <!-- if omega difference is greater than this paraneter, we command low_velocity -->
        <param name="cmd_filtered/global_threshold" type="double" value="1.0"/>   
        <!-- a threshold for stopping the command of low velocity and resuming planner commands -->             
        <param name="cmd_filtered/local_threshold" type="double" value="0.2"/>
        <param name="cmd_filtered/low_velocity" type="double" value = "0.005"/>        
    </node>
</launch>
